org.eclipse.winery.topologymodeler.ui/TmDependencyTestApp/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "tm-dependency-test-app",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build --prod",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular-redux/store": "6.5.7",
+    "@angular/animations": "6.0.4",
+    "@angular/cdk": "6.2.1",
+    "@angular/common": "6.0.4",
+    "@angular/core": "6.0.4",
+    "@angular/cli": "6.0.8",
+    "@angular/compiler": "6.0.4",
+    "@angular/compiler-cli": "6.0.4",
+    "@angular/forms": "6.0.4",
+    "@angular/http": "6.0.4",
+    "@angular/platform-browser": "6.0.4",
+    "@angular/platform-browser-dynamic": "6.0.4",
+    "@angular/router": "6.0.4",
+    "angular2-hotkeys": "2.0.4",
+    "bootstrap": "4.1.1",
+    "core-js": "2.5.6",
+    "css-element-queries": "0.4.0",
+    "element-resize-detector": "1.1.12",
+    "elkjs": "0.0.1",
+    "font-awesome": "4.7.0",
+    "jsplumb": "2.6.12",
```

Comment:
Please update to `2.7.9` to be consistent with the workflowmodeler

Reply:
done